terraform/modules/public-load-balancer/variables.tf

```diff
@@ -0,0 +1,38 @@
+variable "app_domain" {
+  type        = string
+  description = "e.g. test.publishing.service.gov.uk"
+}
+
+variable "app_name" {
+  type        = string
+  description = "A GOV.UK application name. E.g. publisher, content-publisher"
+}
+
+variable "cluster_name" {
+  type    = string
+  default = "govuk" # TODO: Is this the default value?
```

Comment:
I think calling this `cluster_name` could be confusing, because this module shouldn't have anything to do with the ECS cluster. Since this is really a suffix for the name of the ALB that the module creates, we could call it that and add a description saying what it does. I don't know if we settled on terminology for this yet but `workspace_suffix` maybe? (or `name_suffix`?)

I appreciate this whole environment naming thing is a bit of a can of worms though - no worries if we can't solve this neatly in this PR. Might need some more thought about the broader issue of how we pass these names around in a way which is sufficiently clear as to what's what.

Reply:
Thanks, I've changed it to `workspace_suffix` for now. I agree, we may want to revisit this once Roch finishes the multi-cluster work.